_topic_maps/_topic_map.yml

```diff
@@ -1077,7 +1079,7 @@ Topics:
   - Name: Installing the AWS Load Balancer Operator
     File: install-aws-load-balancer-operator
   - Name: Installing the AWS Load Balancer Operator on Secure Token Service cluster
-    File: installing-albo-sts-cluster  
+    File: installing-albo-sts-cluster
```

Comment:
@bergerhoffer does this relate to this PR itself or is this something different ?

Reply:
@kasturinarra This is just trailing whitespace that is automatically removed by my editor (other writers should also have this setting turned on, but as you can see, not everyone does :)

Reply:
ah, got it now, thought you added the entire line. Never mind , thanks !!